core/server/master/src/main/java/alluxio/master/file/FileSystemMasterAuditContext.java

```diff
@@ -35,6 +35,8 @@
   private AuthType mAuthType;
   private String mIp;
   private Inode mSrcInode;
+  private long mCreateTimeMillis;
+  private long mUseTimeMillis;
```

Comment:
I would name them
```suggestion
  private long mCreationTime;
  private long mExecutionTime;
```

Reply:
Ok, I will use System.nanoTime as baolong's suggestion, and only add useTime instead of createTime and useTime beacuse the createTime will be meaningless. The useTime will be named as mExecutionTime